create_dependencies_list.sh

```diff
@@ -26,7 +26,7 @@ fi
 HONO_MAVEN_DEPS="legal/src/main/resources/legal/hono-maven.deps"
 DEPENDENCIES="legal/src/main/resources/legal/DEPENDENCIES"
 
-mvn dependency:list -DexcludeGroupIds=org.eclipse,org.junit -Pjaeger,metrics-prometheus,metrics-graphite,build-docker-image,build-native-image | grep -Poh "\S+:(runtime|compile|provided)" | sed -e 's/^\(.*\)\:.*$/\1/' | sort | uniq > $HONO_MAVEN_DEPS
+mvn dependency:list -DexcludeGroupIds=org.eclipse,org.junit -Pmetrics-prometheus,metrics-graphite,build-docker-image,build-native-image | grep -Poh "\S+:(runtime|compile|provided)" | sed -e 's/^\(.*\)\:.*$/\1/' | sort | uniq > $HONO_MAVEN_DEPS
```

Comment:
IMHO we can remove the `metrics-graphite` profile as well, or do we still need it?